.github/workflows/avh.yml

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Arm Virtual Hardware
+
+on:
+    schedule:
+        - cron: '0 * * * *'
```

Comment:
Why cron, not push/pr events?

Reply:
Virtual devices on each Arm Virtual Hardware (AVH) account currently share the same virtual BLE and (Wi-Fi) IP networks, so running tests from each push/pr in parallel could cause issues if tests are run concurrently. We're open to changing this to push/pr, but this would result in running tests without concurrency for this workflow.

Reply:
I guess the question is this: is anyone going to be watching these jobs to notice when they fail?  The per-PR jobs people notice because their PR fails CI and can't merge.  But if something breaks this job, what happens?

Reply:
And also, the rationale should be added to the yaml as a comment, so someone doesn't "fix" it to run on push/pr.

Reply:
I don't have knowledge in GitHub actions, but sounds like cron is indeed not the right way to solve your concurrent issues @sandeepmistry . Sound like your account point to the same virtual hardware which is a singleton ressource that need to be locked.
It should have something in the spirit of github-action-locks when you googling it..

Reply:
If there's no concerns about having the AVH portion of the test run non-concurrently, I can update the action to run on PR and push and look into using something like github-action-locks or the ["GitHub Actions: Limit workflow run or job concurrency" guide](https://github.blog/changelog/2021-04-19-github-actions-limit-workflow-run-or-job-concurrency/).

For reference the AVH end-to-end test takes ~4 minutes to complete. Ideally, the cross-compile step would run concurrently.

Reply:
I would suggest you look into it indeed. But you can do that on a sub-sequent Pull Request dedicated on this topic

Reply:
Agreed: doing concurrent cross-compile and locking for 4 mins while we run the actual test should be just fine, but better to do it as a followup.

Reply:
Added comment for https://github.com/project-chip/connectedhomeip/pull/25686#discussion_r1137077752 in https://github.com/project-chip/connectedhomeip/pull/25686/commits/e5ca10ceda000a212f4737cbfa31e9e47342f4e5.

Will do a follow up pull request to run this workflow per pull request after this is merged.